packages/mobile/src/services/offline-downloader/offline-download-queue.ts

```diff
@@ -44,7 +44,7 @@ const removeExistingWorkers = () => {
 export const startDownloadWorker = async () => {
   queue.stop()
   queue.configure({
-    concurrency: 10,
+    concurrency: 1,
```

Comment:
how come only 1?

Reply:
It was failing when running in parallel. I think we can track it down and fix it, but wanted to get something up and working for now.

Reply:
It could also be that it just overloaded my internet here. Which is a valid case anyway, but the change is likely not necessary for most users unless we also broke something in the recent changes. Obviously it used to work at 10.

Reply:
sounds good!